mcp-spring/mcp-spring-webflux/src/test/java/io/modelcontextprotocol/WebFluxSseIntegrationTests.java

```diff
@@ -378,9 +380,9 @@ void testRootsServerCloseWithActiveSubscription(String clientType) {
 
 	String emptyJsonSchema = """
 			{
-				"$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
What happened here?

Reply:
A mistake while testing. Btw the $schema filed is optional for the JSON Schema, which explains why it was working.